<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>766</height>
   </rect>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="m_centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="m_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QScrollArea" name="m_options">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>737</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="ImageView2" name="m_imageView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>512</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VideoControls" name="m_videoControls">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>100</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ModuleList" name="m_moduleList" native="true">
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpen0"/>
     <addaction name="actionOpen1"/>
     <addaction name="actionOpen2"/>
     <addaction name="actionOpen3"/>
    </widget>
    <widget class="QMenu" name="menuClose">
     <property name="title">
      <string>Close</string>
     </property>
     <addaction name="actionClose0"/>
     <addaction name="actionClose1"/>
     <addaction name="actionClose2"/>
     <addaction name="actionClose3"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="menuClose"/>
    <addaction name="actionSavePNG"/>
    <addaction name="actionSaveAll"/>
    <addaction name="actionSavePDF"/>
    <addaction name="actionShowInfo"/>
    <addaction name="separator"/>
    <addaction name="actionBatch"/>
    <addaction name="actionBatch2"/>
    <addaction name="actionBatchEx"/>
    <addaction name="actionRecord"/>
    <addaction name="actionRecord2"/>
    <addaction name="separator"/>
    <addaction name="actionModules"/>
    <addaction name="actionSelectDevice"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionReset"/>
    <addaction name="actionFitWidth"/>
    <addaction name="actionFitHeight"/>
    <addaction name="separator"/>
    <addaction name="actionHold"/>
    <addaction name="actionToggle"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen0">
   <property name="text">
    <string>Image/Video #0...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSavePNG">
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionToggle">
   <property name="text">
    <string>Toggle</string>
   </property>
   <property name="shortcut">
    <string>\</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSavePDF">
   <property name="text">
    <string>Save PDF...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionHold">
   <property name="text">
    <string>Hold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionBatch">
   <property name="text">
    <string>Batch...</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record...</string>
   </property>
  </action>
  <action name="actionModules">
   <property name="text">
    <string>Modules...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionRecord2">
   <property name="text">
    <string>Record2...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionBatch2">
   <property name="text">
    <string>Batch2...</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All...</string>
   </property>
  </action>
  <action name="actionFitWidth">
   <property name="text">
    <string>Fit Width</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionFitHeight">
   <property name="text">
    <string>Fit Height</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionOpen1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #1</string>
   </property>
  </action>
  <action name="actionClose0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #0</string>
   </property>
  </action>
  <action name="actionOpen2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #2</string>
   </property>
  </action>
  <action name="actionOpen3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #3</string>
   </property>
  </action>
  <action name="actionClose1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #1</string>
   </property>
  </action>
  <action name="actionClose2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #2</string>
   </property>
  </action>
  <action name="actionClose3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Image/Video #3</string>
   </property>
  </action>
  <action name="actionSelectDevice">
   <property name="text">
    <string>Select CUDA Device...</string>
   </property>
  </action>
  <action name="actionShowInfo">
   <property name="text">
    <string>Show Info...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBatchEx">
   <property name="text">
    <string>BatchEx...</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView2</class>
   <extends>QWidget</extends>
   <header>imageview2.h</header>
   <container>1</container>
   <slots>
    <slot>reset()</slot>
    <slot>zoomIn()</slot>
    <slot>zoomOut()</slot>
    <slot>toggle()</slot>
    <slot>hold()</slot>
    <slot>copy()</slot>
    <slot>fitWidth()</slot>
    <slot>fitHeight()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>VideoControls</class>
   <extends>QFrame</extends>
   <header>videocontrols.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModuleList</class>
   <extends>QWidget</extends>
   <header>modulelist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggle</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHold</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>hold()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFitWidth</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>fitWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFitHeight</sender>
   <signal>triggered()</signal>
   <receiver>m_imageView</receiver>
   <slot>fitHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
